tendrl/gluster_integration/manager/utils.py

```diff
@@ -4,24 +4,18 @@
 import subprocess
 
 LOG = logging.getLogger(__name__)
-TENDRL_CONTEXT = "/etc/tendrl/gluster_integration/tendrl_context"
+TENDRL_CONTEXT = "/var/lib/tendrl/gluster_cluster_id"
```

Comment:
The reason I changed the path was, since its in etc directory it might be considered as a config file by the user and users might meddle with the content. Since this is file is to be written and read only by tendrl(and this need not be exposed to user) felt /var/lib would be right path.

Please provide your thoughts.